spec/services/create_next_edition_service_spec.rb

```diff
@@ -0,0 +1,118 @@
+RSpec.describe CreateNextEditionService do
+  describe ".call" do
+    let(:user) { create :user }
+
+    before do
+      populate_default_government_bulk_data
+    end
+
+    it "aborts if the current edition isn't live" do
+      current_edition = create(:edition, live: false)
+
+      expect { described_class.call(current_edition: current_edition, user: user) }
+        .to raise_error("Can only create a next edition from a live edition")
+    end
+
+    it "updates the documents current edition" do
+      current_edition = create(:edition, :published)
+
+      next_edition = described_class.call(current_edition: current_edition,
+                                          user: user)
+
+      expect(next_edition.current).to be(true)
+    end
+
+    it "returns a draft edition" do
+      current_edition = create(:edition, :published)
+
+      next_edition = described_class.call(current_edition: current_edition,
+                                          user: user)
+
+      expect(next_edition.draft?).to be(true)
+    end
+
+    it "resets change_note, update_type, and propoposed_publish_time values" do
```

Comment:
Is there an extra comma just after update_type?

Reply:
@1pretz1 It's use like this is known as the [Oxford comma](https://www.grammarbook.com/punctuation/commas.asp).  Personally, I don't feel it matters too much unless you have this situation "one, two, three and four, and five" where it adds clarity.  It's a matter of taste I guess - I'm happy to remove it.

Reply:
Ah ha okay didn't know comma's can be used in that way, my grammar is awful tbf so It doesn't surprise me that I don't know this! 😆